frontend/src/utilities/documents.ts

```diff
@@ -119,6 +161,7 @@ registerResponseHandler(ResponseType.ExportDocument, (responseData: Response) =>
 
 registerResponseHandler(ResponseType.SaveDocument, (responseData: Response) => {
 	const saveData = responseData as SaveDocument;
+	state.documents[state.activeDocumentIndex].unsaved = false;
```

Comment:
Let's make this happen in the Rust code, since the frontend isn't supposed to modify state.